chromium_src/chrome/browser/autocomplete/chrome_autocomplete_provider_client.cc

```diff
@@ -4,14 +4,74 @@
 // You can obtain one at https://mozilla.org/MPL/2.0/.
 
 #include "src/chrome/browser/autocomplete/chrome_autocomplete_provider_client.cc"
+
+#include "brave/components/ai_chat/browser/ai_chat_tab_helper.h"
 #include "brave/components/commander/common/buildflags/buildflags.h"
+#include "build/build_config.h"
+
+#if !BUILDFLAG(IS_ANDROID)
+#include "brave/browser/ui/brave_browser.h"
+#include "brave/browser/ui/sidebar/sidebar_controller.h"
+#endif  // BUILDFLAG(!IS_ANDROID)
 
 #if BUILDFLAG(ENABLE_COMMANDER)
 #include "brave/browser/ui/commander/commander_service_factory.h"
 #include "brave/components/commander/browser/commander_frontend_delegate.h"
+#endif  // BUILDFLAG(ENABLE_COMMANDER)
+
+#if BUILDFLAG(ENABLE_AI_CHAT)
+#include "brave/components/ai_chat/common/features.h"
+#include "brave/components/ai_chat/common/pref_names.h"
+#endif  // BUILDFLAG(ENABLE_AI_CHAT)
 
+#if BUILDFLAG(ENABLE_COMMANDER)
 commander::CommanderFrontendDelegate*
 ChromeAutocompleteProviderClient::GetCommanderDelegate() {
   return commander::CommanderServiceFactory::GetForBrowserContext(profile_);
 }
+#endif  // BUILDFLAG(ENABLE_COMMANDER)
+
+#if BUILDFLAG(ENABLE_AI_CHAT)
+void ChromeAutocompleteProviderClient::OpenLeo(const std::u16string& query) {
+#if !BUILDFLAG(IS_ANDROID)
```

Comment:
This could be possible on android @SergeyZhukovsky? It doesn't use side panel, but it is opening the same WebUI via some kind of panel interface.

Reply:
I think so as well, but not 100% sure.